lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -249,22 +249,14 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
     $reportbackItems = $promotedReportbackItems;
   }
 
-  // Check both the global checkbox & node specific one
-  $disable_reactions = dosomething_kudos_disable_reactions($vars['nid']);
-
-  $allowed_tids = [];
-
-  if (!$disable_reactions) {
-    array_push($allowed_tids, dosomething_kudos_get_term_id_by_name('heart'));
-  }
-
   foreach ($reportbackItems as $item) {
-    $item->disable_reactions = $disable_reactions;
-    $item->allowed_reactions = $allowed_tids;
+    $id = $item->id;
     if ($item instanceof ReportbackItem) {
-      $gallery['item_ids'][] = $item->id;
+      $gallery['item_ids'][] = $id;
     }
-    $gallery['items'][] = paraneue_dosomething_get_themed_reportback_item($item);
+
+    $item = paraneue_dosomething_get_themed_reportback_item($item);
```

Comment:
you're overwriting `$item` from the loop with a new var `$item`... it's also not really necessary since you only use it again right below. Just keep it

```php
$gallery['items'][] = paraneue_dosomething_get_themed_reportback_item($item);
```
With `$id` above it, you only use it in one place, so no point in assigning it to a variable, just keep it:

```php
$gallery['item_ids'][] = $item->id;
```